diagnostic_feedback/tasks.py

```diff
@@ -0,0 +1,130 @@
+"""
+Celery task for CSV student answer export.
+"""
```

Comment:
This is obviously very similar to https://github.com/open-craft/problem-builder/blob/master/problem_builder/tasks.py - rather than duplicating the code here, can we make it generic and split it out into xblock-utils or an independent data export block in its own repository?